pandas-stubs/core/indexes/interval.pyi

```diff
@@ -47,37 +69,57 @@ class IntervalIndex(IntervalMixin, ExtensionIndex):
     @classmethod
     def from_tuples(
         cls,
-        data,
+        data: Sequence[tuple[int, int]]
+        | Sequence[tuple[float, float]]
+        | Sequence[tuple[DatetimeLike, DatetimeLike]]
+        | npt.NDArray,
         closed: IntervalClosedType = ...,
         name: Hashable = ...,
         copy: bool = ...,
         dtype: IntervalDtype | None = ...,
     ) -> IntervalIndex: ...
+    def __contains__(self, key: Any) -> bool: ...
     def astype(self, dtype: DtypeArg, copy: bool = ...) -> IntervalIndex: ...
     @property
     def inferred_type(self) -> str: ...
     def memory_usage(self, deep: bool = ...) -> int: ...
+    def is_monotonic_decreasing(self) -> bool: ...
```

Comment:
This is a `property` of `Index`, so I think you can delete this here.  The implementation is different, but we can just inherit from `Index`

Reply:
Removed.